deployment/ccip/changeset/testhelpers/test_helpers_solana_v0_1_0.go

```diff
@@ -2005,11 +2006,12 @@ func Transfer(
 		}
 	case chainsel.FamilySui:
 		msg = SuiSendRequest{
-			Data:         data,
-			Receiver:     common.LeftPadBytes(receiver, 32),
-			ExtraArgs:    extraArgs,
-			FeeToken:     feeToken,
-			TokenAmounts: tokens.([]SuiTokenAmount),
+			Data:             data,
+			Receiver:         common.LeftPadBytes(receiver, 32),
+			ExtraArgs:        extraArgs,
+			FeeToken:         feeToken,
+			TokenAmounts:     tokens.([]SuiTokenAmount),
+			TokenReceiverATA: common.LeftPadBytes(tokenReceiverATA, 32),
```

Comment:
is this just set to empty for now? I don't see any test where this value is non zero